pom.xml

```diff
@@ -49,10 +49,6 @@
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>mailer</artifactId>
         </dependency>
-        <dependency>
-            <groupId>commons-lang</groupId>
-            <artifactId>commons-lang</artifactId>
-        </dependency>
```

Comment:
This dependency is provided by Jenkins core, so it doesn't need to be bundled in the plugin.